elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-5/O/OBWGC2LOKVZWK4SFOZSXE6LUNBUW4ZY.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<identity xmlns="urn:elytron:1.0">
```

Comment:
This file is missing info on how it was generated.

Reply:
Added